server/server.go

```diff
@@ -64,6 +65,7 @@ type Server struct {
 	node          *Node
 	admin         *adminServer
 	status        *statusServer
+	structuredDB  *structured.DB
```

Comment:
I think you need a `structuredServer` similar to the way there is `tsDB` and `tsServer`.